lib/class-wp-rest-widgets-controller.php

```diff
@@ -116,7 +92,7 @@ public function register_routes() {
 			),
 			'schema' => array( $this, 'get_public_item_schema' ),
 		) );
-		register_rest_route( $this->namespace, '/' . $this->rest_base .'/types/(?P<type>[\w-]+)', array(
+		register_rest_route( $this->namespace, '/' . $this->rest_base_types . '/(?P<type>[\w-]+)', array(
```

Comment:
@danielbachhuber So are you suggesting `/wp/v2/widget-type/:type` is preferred?

According to my list above, this will only be used to serve the schema of a specific type with an `OPTIONS` request. Is that normal API design? What should the endpoint serve to a typical `GET` or `POST` request?